lib/utils.js

```diff
@@ -124,34 +124,40 @@ function requestWithRetry(request, options, callback) {
  */
 function authorizedRequestFactory(scopes, config) {
   var authClient;
+  var queuedRequests = [];
```

Comment:
@kjin This type of user-space queing will break debug call-stacks. Consider using a function interface and allowing the event-loop to take of the queueing/dispatching?